### ISSUE TITLE
Chapters no longer included in the m4b file

### ISSUE THREAD

**Issue by HedaConHeda** [rmcrackan/Libation#234]
I updated from Libation 6.7.3 to 6.8.2 and now the Chapters are no longer present in the m4b file. There is still a Menu section in the MediaInfo, but it's empty. The Chapters are still present in the cue file, but I prefer to have all data in a single file.

**Comment by rmcrackan** [comment]
@Mbucari was this a change in the latest AAXClean?

**Comment by HedaConHeda** [comment]
I checked and this bug isn't present in 6.7.8 but it is in 6.8.1.

**Comment by Mbucari** [comment]
Yeah, I just noticed this about an hour ago. Sorry!  Fixed in 0.2.7

**Comment by rmcrackan** [comment]
@HedaConHeda Fixed in [v6.8.4](https://github.com/rmcrackan/Libation/releases/latest). Sorry about that.

@Mbucari Thanks for the quick turn around time!
